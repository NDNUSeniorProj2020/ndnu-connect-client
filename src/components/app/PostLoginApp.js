import React from 'react';
import PropTypes from 'prop-types';

import Navbar from '../Navbar/Navbar';
import Main from '../Main';

export default function PostLoginApp({ handleLogout }) {
	return (
		<div className={'App'}>
			<div className="container">
				<Navbar handleLogout={handleLogout} />
				<Main />
			</div>
		</div>
	);
}

PostLoginApp.propTypes = { handleLogout: PropTypes.func };
PostLoginApp.defaultProps = { handleLogout: f => f };
