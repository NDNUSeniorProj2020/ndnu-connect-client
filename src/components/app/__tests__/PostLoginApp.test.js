import React from 'react';
import { shallow } from 'enzyme';

import PostLoginApp from '../PostLoginApp';

describe('tests for PostLoginApp', () => {
	describe('snapshot tests', () => {
		it('renders without crashing', () => {
			const tree = shallow(<PostLoginApp />);
			expect(tree).toMatchSnapshot();
		});
	});

	describe('unit tests', () => {
		it('has a handleLogout default prop', () => expect(PostLoginApp.defaultProps.handleLogout('f')).toEqual('f'));
	})
});
