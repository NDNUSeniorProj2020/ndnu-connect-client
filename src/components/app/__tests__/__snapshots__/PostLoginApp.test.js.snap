// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`tests for PostLoginApp snapshot tests renders without crashing 1`] = `ShallowWrapper {}`;
